### ISSUE TITLE
Tests hang on silent app errors

### ISSUE THREAD

**Issue by zeppelin** [embermap/ember-cli-fastboot-testing#3]
When [`fastboot.visit()`](https://github.com/embermap/ember-cli-fastboot-testing/blob/master/index.js#L32) fails, the response never gets sent or logged to the console, making the tests hang.

It would be cool if the error was both logged AND sent inside the response body.

**Comment by ryanto** [comment]
Thanks for the report!

Could you give me an example of a silent error? Would love to get a test case coded for this.

**Comment by ryanto** [comment]
Hey @zeppelin - Just wanted to follow up and see if you could point me at an example of this happening. Thanks!

**Comment by zeppelin** [comment]
I'm gonna try to add the addon to the app once again, after I finish the current milestone!

**Comment by zeppelin** [comment]
So one example is when the request fails with

```
Error: The host header did not match a hostWhitelist entry. Host header: ember-cli-fastboot-testing.localhost
```

This error never makes to the console.

**Comment by ryanto** [comment]
Awesome and thanks. This is all fixed now and the error message will be rendered inside of the testing container.

It's released as `0.0.12`
